Index: biology/p5-AcePerl/files/patch-RPC_RPC.xs
===================================================================
--- biology/p5-AcePerl/files/patch-RPC_RPC.xs	(revision 0)
+++ biology/p5-AcePerl/files/patch-RPC_RPC.xs	(working copy)
@@ -0,0 +1,11 @@
+--- RPC/RPC.xs.orig	2014-03-27 15:24:09.000000000 +0000
++++ RPC/RPC.xs	2014-03-27 15:24:26.000000000 +0000
+@@ -154,7 +154,7 @@
+     case '_':
+ 	if (strEQ(name, "_ACECLIENT_"))
+ #ifdef _ACECLIENT_
+-	    return _ACECLIENT_;
++	    return 0;
+ #else
+ 	    goto not_there;
+ #endif
