Index: chinese/gcin/Makefile
===================================================================
--- chinese/gcin/Makefile	(revision 372681)
+++ chinese/gcin/Makefile	(working copy)
@@ -14,8 +14,8 @@
 LICENSE=	LGPL21
 
 HAS_CONFIGURE=	YES
-CONFIGURE_ARGS+=	--use_gtk3=N --use_qt3=N
-USE_GNOME=	gtk20
+CONFIGURE_ARGS+=	--use_qt3=N
+USE_GNOME=	gtk20 gtk30
 USES=		gmake pkgconfig tar:xz
 USE_XORG=	x11 xtst xext
 INSTALLS_ICONS=	YES
