Index: chinese/oxim/Makefile
===================================================================
--- chinese/oxim/Makefile	(revision 355603)
+++ chinese/oxim/Makefile	(working copy)
@@ -80,5 +80,7 @@
 post-patch:
 	${REINPLACE_CMD} 's|%%PREFIX%%|${PREFIX}|' \
 			${WRKSRC}/src/gtk-immodule/gtkintl.h
+	${REINPLACE_CMD} -e '/XFT_LIBS =/s/$$/ -lfontconfig -lXext/' \
+			${WRKSRC}/src/xim/Makefile.in
 
 .include <bsd.port.mk>
