Index: chinese/libchewing/Makefile
===================================================================
--- chinese/libchewing/Makefile	(revision 378936)
+++ chinese/libchewing/Makefile	(working copy)
@@ -11,6 +11,7 @@
 LICENSE=	LGPL21
 LICENSE_FILE=	 ${WRKSRC}/COPYING
 
+BUILD_DEPENDS=	${LOCALBASE}/bin/makeinfo:${PORTSDIR}/print/texinfo
 LIB_DEPENDS=	libsqlite3.so:${PORTSDIR}/databases/sqlite3
 
 USES=		cmake gmake libtool pathfix pkgconfig tar:bzip2
@@ -21,8 +22,10 @@
 GH_TAGNAME=	v${PORTVERSION}
 GH_COMMIT=	6d6e67c
 
+CMAKE_ARGS+=	-DMAKEINFO=${LOCALBASE}/bin/makeinfo
 CFLAGS+=	-I${LOCALBASE}/include
 INFO=		libchewing
+INFO_PATH=	share/info
 
 .include <bsd.port.pre.mk>
 
