Index: chinese/fireflyttf/Makefile
===================================================================
--- chinese/fireflyttf/Makefile	(revision 361960)
+++ chinese/fireflyttf/Makefile	(working copy)
@@ -48,6 +48,7 @@
 
 do-install:
 	${MKDIR} ${STAGEDIR}${FONTSDIR}
+	${MKDIR} ${STAGEDIR}${XFONTSDIR}
 	${INSTALL_DATA} ${WRKSRC}/${TTF} ${STAGEDIR}${FONTSDIR}
 	${INSTALL_DATA} ${FILESDIR}/local.conf-sung ${STAGEDIR}${FONTSDIR}
 .if ${PORT_OPTIONS:MX11}
