Index: chinese/fireflyttf/Makefile
===================================================================
--- chinese/fireflyttf/Makefile	(revision 372681)
+++ chinese/fireflyttf/Makefile	(working copy)
@@ -45,6 +45,7 @@
 
 do-install:
 	@${MKDIR} ${STAGEDIR}${FONTSDIR}
+	@${MKDIR} ${STAGEDIR}${XFONTSDIR}
 	${INSTALL_DATA} ${WRKSRC}/${TTF} ${STAGEDIR}${FONTSDIR}
 	${INSTALL_DATA} ${FILESDIR}/local.conf-sung ${STAGEDIR}${FONTSDIR}
 
