Index: devel/git/pkg-plist
===================================================================
--- devel/git/pkg-plist	(revision 357645)
+++ devel/git/pkg-plist	(working copy)
@@ -753,7 +753,6 @@
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-sha1-array.txt
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-sigchain.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-sigchain.txt
-%%HTMLDOCS%%%%DOCSDIR%%/technical/api-hash.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-merge.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-merge.txt
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-setup.html
@@ -777,7 +776,6 @@
 %%HTMLDOCS%%%%DOCSDIR%%/technical/pack-protocol.txt
 %%HTMLDOCS%%%%DOCSDIR%%/technical/pack-protocol.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-index.txt
-%%HTMLDOCS%%%%DOCSDIR%%/technical/api-hash.txt
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-directory-listing.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-diff.html
 %%HTMLDOCS%%%%DOCSDIR%%/technical/api-in-core-index.txt
@@ -1176,7 +1174,6 @@
 %%HTMLDOCS%%%%DOCSDIR%%/RelNotes/1.9.1.txt
 %%HTMLDOCS%%%%DOCSDIR%%/RelNotes/1.9.2.txt
 %%HTMLDOCS%%%%DOCSDIR%%/RelNotes/1.9.3.txt
-%%HTMLDOCS%%%%DOCSDIR%%/RelNotes/1.9.4.txt
 %%HTMLDOCS%%%%DOCSDIR%%/RelNotes/2.0.0.txt
 %%GITWEB%%%%EXAMPLESDIR%%/gitweb/gitweb.cgi
 %%GITWEB%%%%EXAMPLESDIR%%/gitweb/static/gitweb.js
