Index: devel/libdbusmenu-qt/pkg-plist
===================================================================
--- devel/libdbusmenu-qt/pkg-plist	(revision 357865)
+++ devel/libdbusmenu-qt/pkg-plist	(working copy)
@@ -28,7 +28,6 @@
 %%GENDOCS%%%%DOCSDIR%%/dynsections.js
 %%GENDOCS%%%%DOCSDIR%%/files.html
 %%GENDOCS%%%%DOCSDIR%%/ftv2blank.png
-%%GENDOCS%%%%DOCSDIR%%/ftv2cl.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2doc.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2folderclosed.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2folderopen.png
@@ -36,9 +35,7 @@
 %%GENDOCS%%%%DOCSDIR%%/ftv2link.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2mlastnode.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2mnode.png
-%%GENDOCS%%%%DOCSDIR%%/ftv2mo.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2node.png
-%%GENDOCS%%%%DOCSDIR%%/ftv2ns.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2plastnode.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2pnode.png
 %%GENDOCS%%%%DOCSDIR%%/ftv2splitbar.png
@@ -53,41 +50,9 @@
 %%GENDOCS%%%%DOCSDIR%%/nav_h.png
 %%GENDOCS%%%%DOCSDIR%%/open.png
 %%GENDOCS%%%%DOCSDIR%%/pages.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_61.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_61.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_63.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_63.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_64.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_64.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_69.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_69.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_6d.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_6d.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_73.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_73.js
-%%GENDOCS%%%%DOCSDIR%%/search/all_75.html
-%%GENDOCS%%%%DOCSDIR%%/search/all_75.js
-%%GENDOCS%%%%DOCSDIR%%/search/classes_64.html
-%%GENDOCS%%%%DOCSDIR%%/search/classes_64.js
 %%GENDOCS%%%%DOCSDIR%%/search/close.png
-%%GENDOCS%%%%DOCSDIR%%/search/functions_61.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_61.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_63.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_63.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_64.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_64.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_69.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_69.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_6d.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_6d.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_73.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_73.js
-%%GENDOCS%%%%DOCSDIR%%/search/functions_75.html
-%%GENDOCS%%%%DOCSDIR%%/search/functions_75.js
 %%GENDOCS%%%%DOCSDIR%%/search/mag_sel.png
 %%GENDOCS%%%%DOCSDIR%%/search/nomatches.html
-%%GENDOCS%%%%DOCSDIR%%/search/pages_64.html
-%%GENDOCS%%%%DOCSDIR%%/search/pages_64.js
 %%GENDOCS%%%%DOCSDIR%%/search/search.css
 %%GENDOCS%%%%DOCSDIR%%/search/search.js
 %%GENDOCS%%%%DOCSDIR%%/search/search_l.png
