Index: devel/gitg/Makefile
===================================================================
--- devel/gitg/Makefile	(revision 457506)
+++ devel/gitg/Makefile	(working copy)
@@ -2,10 +2,10 @@
 # $FreeBSD$
 
 PORTNAME=	gitg
-PORTVERSION=	3.24.0
-PORTREVISION=	1
+PORTVERSION=	3.26.0
 CATEGORIES=	devel deskutils gnome
 MASTER_SITES=	GNOME
+DIST_SUBDIR=	gnome3
 
 MAINTAINER=	kwm@FreeBSD.org
 COMMENT=	GTK-based git repository viewer
@@ -13,8 +13,6 @@
 LICENSE=	GPLv2
 LICENSE_FILE=	${WRKSRC}/COPYING
 
-BROKEN=		fails to configure
-
 BUILD_DEPENDS=	gsettings-desktop-schemas>=0:devel/gsettings-desktop-schemas \
 		valac:lang/vala
 LIB_DEPENDS=	libsoup-2.4.so:devel/libsoup \
Index: devel/gitg/distinfo
===================================================================
--- devel/gitg/distinfo	(revision 457506)
+++ devel/gitg/distinfo	(working copy)
@@ -1,3 +1,3 @@
-TIMESTAMP = 1495094129
-SHA256 (gitg-3.24.0.tar.xz) = 3e4ec4a8ae83bc7ced8c7610927ade70e37daa5e8beeb4f357a6ea30b4cc951e
-SIZE (gitg-3.24.0.tar.xz) = 1703132
+TIMESTAMP = 1513152551
+SHA256 (gnome3/gitg-3.26.0.tar.xz) = 26730d437d6a30d6e341b9e8da99d2134dce4b96022c195609f45062f82b54d5
+SIZE (gnome3/gitg-3.26.0.tar.xz) = 1551308
Index: devel/gitg/pkg-plist
===================================================================
--- devel/gitg/pkg-plist	(revision 457506)
+++ devel/gitg/pkg-plist	(working copy)
@@ -73,6 +73,7 @@
 %%NLS%%share/locale/sv/LC_MESSAGES/gitg.mo
 %%NLS%%share/locale/tr/LC_MESSAGES/gitg.mo
 %%NLS%%share/locale/uk/LC_MESSAGES/gitg.mo
+%%NLS%%share/locale/vi/LC_MESSAGES/gitg.mo
 %%NLS%%share/locale/zh_CN/LC_MESSAGES/gitg.mo
 %%NLS%%share/locale/zh_HK/LC_MESSAGES/gitg.mo
 %%NLS%%share/locale/zh_TW/LC_MESSAGES/gitg.mo
