Index: devel/py-dbus/pkg-plist
===================================================================
--- devel/py-dbus/pkg-plist	(revision 352106)
+++ devel/py-dbus/pkg-plist	(working copy)
@@ -72,7 +72,6 @@
 %%PORTDOCS%%%%DOCSDIR%%/PY3PORT.txt
 %%PORTDOCS%%%%DOCSDIR%%/README
 %%HTML%%%%PORTDOCS%%%%DOCSDIR%%/README.html
-%%HTML%%%%PORTDOCS%%%%DOCSDIR%%/TODO.html
 %%PORTDOCS%%%%DOCSDIR%%/tutorial.txt
 %%HTML%%%%PORTDOCS%%%%DOCSDIR%%/tutorial.html
 %%PORTEXAMPLES%%%%EXAMPLESDIR%%/example-async-client.py
