Index: devel/valgrind/Makefile
===================================================================
--- devel/valgrind/Makefile	(revision 519661)
+++ devel/valgrind/Makefile	(working copy)
@@ -82,8 +82,10 @@
 PLIST_SUB+=	AMD64="@comment "
 .endif
 
+post-patch:
+	${REINPLACE_CMD} -e 's|ifdef NT_GNU_BUILD_ID|if 0|' \
+		${WRKSRC}/coregrind/m_debuginfo/readelf.c
 .if !${PORT_OPTIONS:MDOCS}
-post-patch:
 	@${RM} -r ${WRKSRC}/docs/html
 .endif
 
