Index: devel/libthai/pkg-plist
===================================================================
--- devel/libthai/pkg-plist	(revision 457506)
+++ devel/libthai/pkg-plist	(working copy)
@@ -29,9 +29,7 @@
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_21d789961216af290d042750153b8196.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_27634488b513a0cc2afc6ccdc5595e06.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_2b8aac3db400906f5e79e7f370615f3e.html
-%%DOXYGEN%%%%DOCSDIR%%/html/dir_59425e443f801f1f2fd8bbe4959a3ccf.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_640b03dcaffdb7a9c0cde9fd2f0ca223.html
-%%DOXYGEN%%%%DOCSDIR%%/html/dir_68267d1309a1af8e8297ef4c3efbcdba.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_7f62d35e0bc619b4325d379e099523b3.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_8c685ef5a7dbf790469f6c666e21f413.html
 %%DOXYGEN%%%%DOCSDIR%%/html/dir_98a1b52be5bcdc51cfeaab671abbf84f.html
