Index: devel/pydbus-common/Makefile
===================================================================
--- devel/pydbus-common/Makefile	(revision 504425)
+++ devel/pydbus-common/Makefile	(working copy)
@@ -35,7 +35,7 @@
 HTMLDOCS_DESC=	build html documentation
 HTMLDOCS_IMPLIES=	DOCS
 HTMLDOCS_CONFIGURE_ENABLE=	documentation
-HTMLDOCS_BUILD_DEPENDS=		sphinx-build:textproc/py-sphinx
+HTMLDOCS_BUILD_DEPENDS=		py27-sphinx>0:textproc/py-sphinx@py27
 
 post-patch-DOCS-on:
 	@${REINPLACE_CMD} -e 's|HEADERS install-dist_docDATA|HEADERS|g ; \
