Index: editors/libreoffice/Makefile
===================================================================
--- editors/libreoffice/Makefile	(revision 385165)
+++ editors/libreoffice/Makefile	(working copy)
@@ -243,7 +243,8 @@
 		eeb2c7ddf0d302fba4bfc6e97eac9624-libbase-1.1.6.zip:src \
 		f94d9870737518e3b597f9265f4e9803-libserializer-1.1.6.zip:src
 JAVA_BUILD_DEPENDS=	ant:${PORTSDIR}/devel/apache-ant \
-			${JAVAJARDIR}/junit.jar:${PORTSDIR}/java/junit
+			${JAVAJARDIR}/junit.jar:${PORTSDIR}/java/junit \
+			${JAVAJARDIR}/hamcrest.jar:${PORTSDIR}/java/hamcrest
 # XXX jni.h from GNU classpath causes ABI conflicts.
 JAVA_CONFLICTS_BUILD=	classpath-[0-9]*
 JAVA_CONFIGURE_ON=	--enable-ext-wiki-publisher \
@@ -250,7 +251,8 @@
 			--enable-report-builder \
 			--with-ant-home=${LOCALBASE}/share/java/apache-ant \
 			--with-jdk-home="${JAVA_HOME}" \
-			--with-junit=${JAVAJARDIR}/junit.jar
+			--with-junit=${JAVAJARDIR}/junit.jar \
+			--with-hamcrest=${JAVAJARDIR}/hamcrest.jar
 JAVA_CONFIGURE_OFF=	--without-java
 JAVA_BUILD=	yes
 JAVA_USE=	JAVA=yes
