Index: editors/libreoffice/Makefile
===================================================================
--- editors/libreoffice/Makefile	(revision 366222)
+++ editors/libreoffice/Makefile	(working copy)
@@ -251,7 +251,7 @@
 MMEDIA_USE=	GSTREAMER=yes
 MMEDIA_CONFIGURE_ENABLE=	gstreamer-0-10
 
-PGSQL_USES=	pgsql:90+
+PGSQL_USES=	pgsql
 PGSQL_CONFIGURE_ENABLE=	postgresql-sdbc
 PGSQL_CONFIGURE_WITH=	gssapi krb5
 
