Index: editors/emacs/Makefile
===================================================================
--- editors/emacs/Makefile	(revision 533654)
+++ editors/emacs/Makefile	(working copy)
@@ -176,7 +176,7 @@
 M17N_IMPLIES=			XFT
 M17N_LIB_DEPENDS=		libm17n.so:devel/m17n-lib
 M17N_CONFIGURE_WITH=		m17n-flt
-MAGICK_LIB_DEPENDS=		libMagickCore-7.so:graphics/ImageMagick7
+MAGICK_LIB_DEPENDS=		libMagickCore-6.so:graphics/ImageMagick6
 MAGICK_CONFIGURE_WITH=		imagemagick
 MAILUTILS_LIB_DEPENDS=		libmailutils.so:mail/mailutils
 MAILUTILS_CONFIGURE_WITH=	mailutils
