Index: emulators/bochs/Makefile
===================================================================
--- emulators/bochs/Makefile	(revision 378936)
+++ emulators/bochs/Makefile	(working copy)
@@ -264,6 +264,7 @@
 .endfor
 .if ${PORT_OPTIONS:MPLUGINS}
 	@${FIND} ${STAGEDIR}${PREFIX}/lib/bochs ! -type d | \
+		${SED} '/\.la$$//d' | \
 		${SED} 's,^${STAGEDIR}${PREFIX}/,,' >> ${TMPPLIST}
 	@${FIND} ${STAGEDIR}${PREFIX}/lib/bochs -type d -empty | ${SORT} -r | \
 		${SED} 's,^${STAGEDIR}${PREFIX}/,@dir ,' >> ${TMPPLIST}
