Index: Makefile
===================================================================
--- Makefile	(revision 431453)
+++ Makefile	(working copy)
@@ -65,7 +65,7 @@
 STATIC_LINK_PREVENTS=		GTK2 X11
 STATIC_LINK_PREVENTS_MSG=	X11 ui cannot be built static
 NCURSES_USES=		ncurses:base
-SAMBA_RUN_DEPENDS=	${LOCALBASE}/sbin/smbd:net/samba36
+SAMBA_RUN_DEPENDS=      ${LOCALBASE}/sbin/smbd:net/samba43
 DOCS_BUILD_DEPENDS=	texi2html:textproc/texi2html
 DOCS_USES=		makeinfo
 DOCS_MAKE_ARGS_OFF=	NOPORTDOCS=1
