Index: emulators/qemu-devel/Makefile
===================================================================
--- emulators/qemu-devel/Makefile	(revision 406159)
+++ emulators/qemu-devel/Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	qemu
 DISTVERSION=	2.5.0
+PORTREVISION=	0.1
 CATEGORIES=	emulators
 MASTER_SITES=	http://wiki.qemu.org/download/
 PKGNAMESUFFIX=	-devel
Index: emulators/qemu-devel/pkg-plist
===================================================================
--- emulators/qemu-devel/pkg-plist	(revision 406159)
+++ emulators/qemu-devel/pkg-plist	(working copy)
@@ -4,34 +4,34 @@
 bin/qemu-img
 bin/qemu-io
 bin/qemu-nbd
-%%X86_TARGETS%%bin/qemu-system-aarch64
-%%X86_TARGETS%%bin/qemu-system-alpha
-%%X86_TARGETS%%bin/qemu-system-arm
-%%X86_TARGETS%%bin/qemu-system-cris
+%%NO_X86_TARGETS%%bin/qemu-system-aarch64
+%%NO_X86_TARGETS%%bin/qemu-system-alpha
+%%NO_X86_TARGETS%%bin/qemu-system-arm
+%%NO_X86_TARGETS%%bin/qemu-system-cris
 bin/qemu-system-i386
-%%X86_TARGETS%%bin/qemu-system-lm32
-%%X86_TARGETS%%bin/qemu-system-m68k
-%%X86_TARGETS%%bin/qemu-system-microblaze
-%%X86_TARGETS%%bin/qemu-system-microblazeel
-%%X86_TARGETS%%bin/qemu-system-mips
-%%X86_TARGETS%%bin/qemu-system-mips64
-%%X86_TARGETS%%bin/qemu-system-mips64el
-%%X86_TARGETS%%bin/qemu-system-mipsel
-%%X86_TARGETS%%bin/qemu-system-moxie
-%%X86_TARGETS%%bin/qemu-system-or32
-%%X86_TARGETS%%bin/qemu-system-ppc
-%%X86_TARGETS%%bin/qemu-system-ppc64
-%%X86_TARGETS%%bin/qemu-system-ppcemb
-%%X86_TARGETS%%bin/qemu-system-s390x
-%%X86_TARGETS%%bin/qemu-system-sh4
-%%X86_TARGETS%%bin/qemu-system-sh4eb
-%%X86_TARGETS%%bin/qemu-system-sparc
-%%X86_TARGETS%%bin/qemu-system-sparc64
-%%X86_TARGETS%%bin/qemu-system-tricore
-%%X86_TARGETS%%bin/qemu-system-unicore32
+%%NO_X86_TARGETS%%bin/qemu-system-lm32
+%%NO_X86_TARGETS%%bin/qemu-system-m68k
+%%NO_X86_TARGETS%%bin/qemu-system-microblaze
+%%NO_X86_TARGETS%%bin/qemu-system-microblazeel
+%%NO_X86_TARGETS%%bin/qemu-system-mips
+%%NO_X86_TARGETS%%bin/qemu-system-mips64
+%%NO_X86_TARGETS%%bin/qemu-system-mips64el
+%%NO_X86_TARGETS%%bin/qemu-system-mipsel
+%%NO_X86_TARGETS%%bin/qemu-system-moxie
+%%NO_X86_TARGETS%%bin/qemu-system-or32
+%%NO_X86_TARGETS%%bin/qemu-system-ppc
+%%NO_X86_TARGETS%%bin/qemu-system-ppc64
+%%NO_X86_TARGETS%%bin/qemu-system-ppcemb
+%%NO_X86_TARGETS%%bin/qemu-system-s390x
+%%NO_X86_TARGETS%%bin/qemu-system-sh4
+%%NO_X86_TARGETS%%bin/qemu-system-sh4eb
+%%NO_X86_TARGETS%%bin/qemu-system-sparc
+%%NO_X86_TARGETS%%bin/qemu-system-sparc64
+%%NO_X86_TARGETS%%bin/qemu-system-tricore
+%%NO_X86_TARGETS%%bin/qemu-system-unicore32
 bin/qemu-system-x86_64
-%%X86_TARGETS%%bin/qemu-system-xtensa
-%%X86_TARGETS%%bin/qemu-system-xtensaeb
+%%NO_X86_TARGETS%%bin/qemu-system-xtensa
+%%NO_X86_TARGETS%%bin/qemu-system-xtensaeb
 man/man1/qemu.1.gz
 man/man1/qemu-img.1.gz
 man/man8/qemu-ga.8.gz
