Index: ftp/bsdftpd-ssl/Makefile
===================================================================
--- ftp/bsdftpd-ssl/Makefile	(revision 416870)
+++ ftp/bsdftpd-ssl/Makefile	(working copy)
@@ -11,7 +11,6 @@
 MAINTAINER=	skynick@mail.sc.ru
 COMMENT=	Secure FTP client/server with TLS/SSL support
 
-BROKEN=		unfetchable
 DEPRECATED=	Broken for more than 6 months
 EXPIRATION_DATE=	2016-07-04
 USE_OPENSSL=	yes
