diff --git a/games/gnome-mahjongg/Makefile b/games/gnome-mahjongg/Makefile
index cf7b515c05db..3da00b341d3e 100644
--- a/games/gnome-mahjongg/Makefile
+++ b/games/gnome-mahjongg/Makefile
@@ -15,9 +15,11 @@ LICENSE_FILE=	${WRKSRC}/COPYING
 BUILD_DEPENDS=	vala:lang/vala \
 		itstool:textproc/itstool
 
+USES=		gettext gmake gnome localbase meson pathfix pkgconfig tar:xz
+
 PORTSCOUT=	limitw:1,even
 
-USES=		gettext gmake gnome localbase meson pathfix pkgconfig tar:xz
+CONFIGURE_ARGS=	-Dcompile-schemas=disabled -Dupdate-icon-cache=disabled
 INSTALLS_ICONS=	yes
 USE_GNOME=	cairo gtk30 intlhack librsvg2 libxml2:build
 
