Index: graphics/vips/Makefile
===================================================================
--- graphics/vips/Makefile	(revision 534702)
+++ graphics/vips/Makefile	(working copy)
@@ -112,9 +112,9 @@
 
 .if ${PORT_OPTIONS:MIMAGEMAGICK}
 . if ${PORT_OPTIONS:MX11}
-LIB_DEPENDS+=		libMagickCore-7.so:graphics/ImageMagick7
+LIB_DEPENDS+=		libMagickCore-6.so:graphics/ImageMagick6
 . else
-LIB_DEPENDS+=		libMagickCore-7.so:graphics/ImageMagick7-nox11
+LIB_DEPENDS+=		libMagickCore-6.so:graphics/ImageMagick6-nox11
 . endif
 .elif ${PORT_OPTIONS:MGRAPHMAGICK}
 LIB_DEPENDS+=		libGraphicsMagick.so:graphics/GraphicsMagick
