Index: java/jikes/Makefile
===================================================================
--- java/jikes/Makefile	(revision 477159)
+++ java/jikes/Makefile	(working copy)
@@ -16,7 +16,7 @@
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	ac_cv_search_iconv=${ICONV_LIB} \
 		ac_cv_search_libiconv_open=no
-USES=		iconv gmake tar:bzip2
+USES=		autoreconf iconv gmake tar:bzip2
 
 CPPFLAGS+=	-I${LOCALBASE}/include
 LIBS+=		-L${LOCALBASE}/lib
