Index: java/eclipse/files/patch-eclipse-platform-parent
===================================================================
--- java/eclipse/files/patch-eclipse-platform-parent	(revision 520097)
+++ java/eclipse/files/patch-eclipse-platform-parent	(working copy)
@@ -41,3 +41,12 @@
              </environment>
              <environment>
                <os>win32</os>
+@@ -657,7 +657,7 @@
+           For maintenance streams should always be "M-builds".
+           Ideally, this value would be provided by the environment, see bug 489789.
+         -->
+-        <eclipse-p2-repo.url>https://download.eclipse.org/eclipse/updates/4.12-I-builds</eclipse-p2-repo.url>
++        <eclipse-p2-repo.url>https://download.eclipse.org/eclipse/updates/4.12</eclipse-p2-repo.url>
+       </properties>
+       <repositories>
+         <repository>
