Index: java/junit/Makefile
===================================================================
--- java/junit/Makefile	(revision 385165)
+++ java/junit/Makefile	(working copy)
@@ -2,27 +2,20 @@
 # $FreeBSD$
 
 PORTNAME=	junit
-PORTVERSION=	4.11
-PORTREVISION=	1
+PORTVERSION=	4.12
 CATEGORIES=	java devel
-MASTER_SITES=	GHL
-DISTNAME=	${PORTNAME}${PORTVERSION}
+MASTER_SITES=	http://repo1.maven.org/maven2/${PORTNAME}/${PORTNAME}/${PORTVERSION}/
+DISTFILES=	${PORTNAME}-${PORTVERSION}.jar ${PORTNAME}-${PORTVERSION}-sources.jar
+EXTRACT_ONLY=
 
 MAINTAINER=	danilo@FreeBSD.org
 COMMENT=	Regression testing utility for use with the Java(TM) Language
 
-USE_GITHUB=	yes
-GH_ACCOUNT=	junit-team
-GH_PROJECT=	${PORTNAME}
-GH_TAGNAME=	r${PORTVERSION}
-GH_COMMIT=	c62e2df
+LIB_DEPENDS=	${JAVAJARDIR}/hamcrest.jar:${PORTSDIR}/java/hamcrest
 
-USES=		perl5
 USE_JAVA=	yes
-USE_ANT=	yes
+NO_BUILD=	yes
 
-PORTDOCS=	README.html cpl-v10.html doc javadoc
-PORTEXAMPLES=	*
 DATADIR=	${JAVASHAREDIR}/${PORTNAME}
 PLIST_FILES=	%%JAVAJARDIR%%/junit.jar \
 		%%JAVAJARDIR%%/junit4.jar \
@@ -29,18 +22,11 @@
 		%%DATADIR%%/src.jar
 
 do-install:
-	${INSTALL_DATA} ${INSTALL_WRKSRC}/junit${PORTVERSION}-SNAPSHOT/junit-${PORTVERSION}-SNAPSHOT.jar \
+	${INSTALL_DATA} ${DISTDIR}/junit-${PORTVERSION}.jar \
 		 ${STAGEDIR}${JAVAJARDIR}/junit.jar
 	${LN} -sf junit.jar ${STAGEDIR}${JAVAJARDIR}/junit4.jar
 	@${MKDIR} ${STAGEDIR}${DATADIR}
-	${INSTALL_DATA} ${INSTALL_WRKSRC}/junit${PORTVERSION}-SNAPSHOT/junit-${PORTVERSION}-SNAPSHOT-src.jar \
+	${INSTALL_DATA} ${DISTDIR}/junit-${PORTVERSION}-sources.jar \
 		${STAGEDIR}${DATADIR}/src.jar
-	cd ${INSTALL_WRKSRC}/junit${PORTVERSION}-SNAPSHOT/junit/samples \
-		&& ${COPYTREE_SHARE} . ${STAGEDIR}${EXAMPLESDIR}/
-	cd ${INSTALL_WRKSRC} \
-		&& ${FIND} ${PORTDOCS:Njavadoc} -type d -exec ${MKDIR} ${STAGEDIR}${DOCSDIR}/\{} \; \
-		&& ${FIND} ${PORTDOCS:Njavadoc} -type f -exec ${INSTALL_DATA} \{} ${STAGEDIR}${DOCSDIR}/\{} \;
-	cd ${INSTALL_WRKSRC}/junit${PORTVERSION}-SNAPSHOT \
-		&& ${FIND} javadoc -type d -exec ${MKDIR} ${STAGEDIR}${DOCSDIR}/\{} \; \
 
 .include <bsd.port.mk>
Index: java/junit/distinfo
===================================================================
--- java/junit/distinfo	(revision 385165)
+++ java/junit/distinfo	(working copy)
@@ -1,2 +1,4 @@
-SHA256 (junit4.11.tar.gz) = a9266060d72c43411c93d1bfae952134ad91f4115bf5769e9a0d17b3d51bd097
-SIZE (junit4.11.tar.gz) = 1974279
+SHA256 (junit-4.12.jar) = 59721f0805e223d84b90677887d9ff567dc534d7c502ca903c0c2b17f05c116a
+SIZE (junit-4.12.jar) = 314932
+SHA256 (junit-4.12-sources.jar) = 9f43fea92033ad82bcad2ae44cec5c82abc9d6ee4b095cab921d11ead98bf2ff
+SIZE (junit-4.12-sources.jar) = 200355
