Index: lang/julia/Makefile
===================================================================
--- lang/julia/Makefile	(revision 435057)
+++ lang/julia/Makefile	(working copy)
@@ -4,7 +4,7 @@
 PORTNAME=	julia
 PORTVERSION=	0.5.0
 DISTVERSIONSUFFIX=	-full
-PORTREVISION=	3
+PORTREVISION=	4
 CATEGORIES=	lang math
 MASTER_SITES=	https://github.com/JuliaLang/julia/releases/download/v${PORTVERSION}/
 
@@ -20,8 +20,7 @@
 		libgit2.so:devel/libgit2 \
 		libgmp.so:math/gmp \
 		libmpfr.so:math/mpfr \
-		libpcre2-8.so:devel/pcre2 \
-		libarpack.so:math/arpack-ng
+		libpcre2-8.so:devel/pcre2
 BUILD_DEPENDS=	llvm-config38:devel/llvm38 \
 		pcre2-config:devel/pcre2 \
 		patchelf:sysutils/patchelf
@@ -43,8 +42,12 @@
 		USE_GPL_LIBS=${USE_GPL_LIBS}
 
 OPTIONS_DEFINE=	EXAMPLES DEBUG DOCS GPL_LIBS DESKTOP NATIVE
+OPTIONS_GROUP=	PRIVATE
+OPTIONS_GROUP_PRIVATE=	ARPACK
 OPTIONS_SUB=	yes
 
+PRIVATE_DESC=	Build self-shipped private depends
+
 DEBUG_VARS=	FORCE_ASSERTIONS=1 \
 		ALL_TARGET=all
 
@@ -79,6 +82,14 @@
 .endif
 .endif
 
+ARPACK_DESC=	Build self-shipped private arpack-ng
+.if ${PORT_OPTIONS:MARPACK}
+MAKE_ARGS+=	USE_SYSTEM_ARPACK=0
+.else
+MAKE_ARGS+=	USE_SYSTEM_ARPACK=1
+LIB_DEPENDS+=	libarpack.so:math/arpack-ng
+.endif
+
 .if ${ARCH} == "i386"
 EXTRA_PATCHES+=	${PATCHDIR}/extra-patch-ui_Makefile
 .endif
Index: lang/julia/files/patch-Make.user
===================================================================
--- lang/julia/files/patch-Make.user	(revision 435057)
+++ lang/julia/files/patch-Make.user	(working copy)
@@ -1,7 +1,6 @@
---- Make.user.orig	2016-12-17 12:39:14 UTC
+--- Make.user.orig	2017-02-26 09:48:03 UTC
 +++ Make.user
-@@ -0,0 +1,25 @@
-+USE_SYSTEM_ARPACK=	1
+@@ -0,0 +1,24 @@
 +USE_SYSTEM_BLAS=	1
 +USE_SYSTEM_GMP=		1
 +USE_SYSTEM_LAPACK=	1
Index: lang/julia/pkg-plist
===================================================================
--- lang/julia/pkg-plist	(revision 435057)
+++ lang/julia/pkg-plist	(working copy)
@@ -29,6 +29,8 @@
 include/julia/uv-unix.h
 include/julia/uv-version.h
 include/julia/uv.h
+%%ARPACK%%lib/julia/libarpack.so
+%%ARPACK%%lib/julia/libarpack.so.2
 lib/julia/libccalltest.so
 lib/julia/libdSFMT.so
 lib/julia/libopenspecfun.so
