Index: lang/hugs/Makefile
===================================================================
--- lang/hugs/Makefile	(revision 355603)
+++ lang/hugs/Makefile	(working copy)
@@ -18,6 +18,7 @@
 #OPTIONS=	HUGS_OPENGL "OpenGL/GLUT support" off
 HUGS_DATE=	Sep2006
 USES=		bison gmake
+USE_GCC=	any
 GNU_CONFIGURE=	yes
 #CONFIGURE_ARGS+=	--prefix=${PREFIX} --enable-ffi
 #CONFIGURE_ENV+=	CC="${CC}" CFLAGS="${CFLAGS}"
