Index: lang/gcc/Makefile
===================================================================
--- lang/gcc/Makefile	(revision 438568)
+++ lang/gcc/Makefile	(working copy)
@@ -169,10 +169,5 @@
 	fi
 .endfor
 	cd ${WRKDIR} ; ${SED} -i -e "/PLIST.lib/ r PLIST.lib" ${TMPPLIST}
-	# This is the canonical GCC port, so add key commands without
-	# version numbers as part of their names.
-.for c in gfortran g++ gcc
-	    ${LN} -s ${c}${SUFFIX} ${STAGEDIR}${PREFIX}/bin/${c}
-.endfor
 
 .include <bsd.port.post.mk>
Index: lang/gcc/pkg-plist
===================================================================
--- lang/gcc/pkg-plist	(revision 438568)
+++ lang/gcc/pkg-plist	(working copy)
@@ -8,9 +8,7 @@
 bin/%%GNU_HOST%%-gfortran%%SUFFIX%%
 bin/c++%%SUFFIX%%
 bin/cpp%%SUFFIX%%
-bin/g++
 bin/g++%%SUFFIX%%
-bin/gcc
 bin/gcc%%SUFFIX%%
 bin/gcc-ar%%SUFFIX%%
 bin/gcc-nm%%SUFFIX%%
@@ -17,7 +15,6 @@
 bin/gcc-ranlib%%SUFFIX%%
 bin/gcov%%SUFFIX%%
 bin/gcov-tool%%SUFFIX%%
-bin/gfortran
 bin/gfortran%%SUFFIX%%
 @comment info/gcc%%SUFFIX%%/dir
 man/man1/cpp%%SUFFIX%%.1.gz
