Index: lang/ruby23/Makefile
===================================================================
--- lang/ruby23/Makefile	(revision 475071)
+++ lang/ruby23/Makefile	(working copy)
@@ -32,7 +32,8 @@
 		--enable-shared \
 		--with-ruby-version=minor \
 		--with-sitedir="${PREFIX}/lib/ruby/site_ruby" \
-		--with-vendordir="${PREFIX}/lib/ruby/vendor_ruby"
+		--with-vendordir="${PREFIX}/lib/ruby/vendor_ruby" \
+		ac_cv_lib_unwind_unw_backtrace=no
 CONFIGURE_ENV=	debugflags=
 CPPFLAGS+=	-I${LOCALBASE}/include
 GNU_CONFIGURE=	yes
