Index: Makefile
===================================================================
--- Makefile	(revision 429389)
+++ Makefile	(working copy)
@@ -39,7 +39,8 @@
 CXXFLAGS+=	-stdlib=libc++ -std=c++11
 MAKE_ARGS+=	prefix=${PREFIX} JCXXFLAGS="${CXXFLAGS}" \
 		FORCE_ASSERTIONS=${FORCE_ASSERTIONS} \
-		USE_GPL_LIBS=${USE_GPL_LIBS}
+		USE_GPL_LIBS=${USE_GPL_LIBS} \
+		CC="${CC}" CXX="${CXX}" FC="${FC}"
 
 OPTIONS_DEFINE=	EXAMPLES DEBUG DOCS GPL_LIBS DESKTOP
 OPTIONS_SUB=	yes
