diff --git a/lang/gcc13-devel/pkg-plist b/lang/gcc13-devel/pkg-plist
index cada193c9907..b9bd0e27dcef 100644
--- a/lang/gcc13-devel/pkg-plist
+++ b/lang/gcc13-devel/pkg-plist
@@ -18,8 +18,6 @@ bin/gcov-dump%%SUFFIX%%
 bin/gcov-tool%%SUFFIX%%
 bin/gfortran%%SUFFIX%%
 bin/lto-dump%%SUFFIX%%
-include/libgccjit++.h
-include/libgccjit.h
 man/man1/cpp%%SUFFIX%%.1.gz
 man/man1/g++%%SUFFIX%%.1.gz
 man/man1/gcc%%SUFFIX%%.1.gz
