Index: lang/ruby23/Makefile
===================================================================
--- lang/ruby23/Makefile	(revision 434884)
+++ lang/ruby23/Makefile	(working copy)
@@ -55,7 +55,7 @@
 EDIT_DESC=		Which line editing lib to use
 LIBEDIT_DESC=		Use libedit
 READLINE_DESC=		Use libreadline
-CAPIDOCS_BUILD_DEPENDS=	doxygen>1.8.12,2:devel/doxygen \
+CAPIDOCS_BUILD_DEPENDS=	doxygen>=1.8.12,2:devel/doxygen \
 			dot:graphics/graphviz
 CAPIDOCS_CONFIGURE_ENABLE=	install-capi
 GMP_CONFIGURE_WITH=	gmp
