Index: lang/ruby24/pkg-plist
===================================================================
--- lang/ruby24/pkg-plist	(revision 438568)
+++ lang/ruby24/pkg-plist	(working copy)
@@ -1863,8 +1863,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d48/nkf_8h__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d48/nkf_8h__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d48/nkf_8h__incl.png
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d49/etc_2constdefs_8h.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d49/etc_2constdefs_8h_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d49/return__sc_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d49/return__sc_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/d4f/structtest__structure__4.html
@@ -1949,8 +1947,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/db4/fiddle_8c__incl.png
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc0/closure__simple_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc0/closure__simple_8c_source.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc4/constdefs_8h.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc4/constdefs_8h_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc5/continuation_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc5/continuation_8c__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d4/dc5/continuation_8c__incl.png
@@ -2194,8 +2190,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dca/strscan_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dca/strscan_8c__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dca/strscan_8c__incl.png
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dd1/socket_2constdefs_8h.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dd1/socket_2constdefs_8h_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dd7/ossl__config_8h.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dd7/ossl__config_8h_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d5/dd9/reader_8c.html
@@ -2337,9 +2331,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d60/nkf_2nkf-utf8_2config_8h__dep__incl.png
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d61/float3_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d61/float3_8c_source.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d64/etc_2constdefs_8h__dep__incl.map
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d64/etc_2constdefs_8h__dep__incl.md5
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d64/etc_2constdefs_8h__dep__incl.png
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d69/structstruct__108byte.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d6a/proc_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d6/d6a/proc_8c__incl.md5
@@ -2885,8 +2876,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/dab/cls__18byte_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/dab/cls__19byte_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/dab/cls__19byte_8c_source.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/dac/openssl_2conftest_8c.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/dac/openssl_2conftest_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/db0/defines_8h.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/db0/defines_8h_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d8/db2/ossl__rand_8h__dep__incl.map
@@ -3016,9 +3005,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d32/tcpserver_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d34/resolv_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d34/resolv_8c_source.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d34/socket_2constdefs_8h__dep__incl.map
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d34/socket_2constdefs_8h__dep__incl.md5
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d34/socket_2constdefs_8h__dep__incl.png
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d3a/cls__12byte_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d3a/cls__12byte_8c__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/d9/d3a/cls__12byte_8c__incl.png
@@ -3379,9 +3365,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dda/generator_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dda/generator_8c__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dda/generator_8c__incl.png
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/ddc/constdefs_8h__dep__incl.map
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/ddc/constdefs_8h__dep__incl.md5
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/ddc/constdefs_8h__dep__incl.png
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dde/id__table_8h__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dde/id__table_8h__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/da/dde/id__table_8h__incl.png
@@ -3734,8 +3717,6 @@
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9c/openssl__missing_8c__incl.map
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9c/openssl__missing_8c__incl.md5
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9c/openssl__missing_8c__incl.png
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9d/socket_2conftest_8c.html
-%%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9d/socket_2conftest_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9f/mips_2ffi_8c.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/d9f/mips_2ffi_8c_source.html
 %%CAPIDOCS%%%%RUBY_DOCDIR%%/capi/html/dc/da4/regenc_8h__incl.map
