Index: mail/fetchmail/Makefile
===================================================================
--- mail/fetchmail/Makefile	(revision 352106)
+++ mail/fetchmail/Makefile	(working copy)
@@ -15,6 +15,7 @@
 LICENSE_COMB=	dual
 
 RUN_DEPENDS=	${LOCALBASE}/share/certs/ca-root-nss.crt:${PORTSDIR}/security/ca_root_nss
+BUILD_DEPENDS=	${LOCALBASE}/bin/python2.7:${PORTSDIR}/lang/python27
 
 # Note USERS can only contain a single word as parts below rely on that.
 USERS=		${PORTNAME}
