Index: mail/fetchmail/Makefile
===================================================================
--- mail/fetchmail/Makefile	(revision 390644)
+++ mail/fetchmail/Makefile	(working copy)
@@ -59,7 +59,7 @@
 .endif
 
 .if ${PORT_OPTIONS:MX11}
-USES+=		python:run
+USES+=		python
 RUN_DEPENDS+=	${PYTHON_PKGNAMEPREFIX}tkinter>=0:${PORTSDIR}/x11-toolkits/py-tkinter
 PLIST_SUB+=	NOX11="@comment "
 .else
