Index: math/py-sympy/Makefile
===================================================================
--- math/py-sympy/Makefile	(revision 457506)
+++ math/py-sympy/Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	sympy
 PORTVERSION=	1.1.1
+PORTREVISION=	1
 CATEGORIES=	math python
 MASTER_SITES=	CHEESESHOP
 PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
@@ -15,7 +16,7 @@
 RUN_DEPENDS=	${PYTHON_PKGNAMEPREFIX}mpmath>=0.19:math/py-mpmath@${FLAVOR}
 
 USES=		python
-USE_PYTHON=	autoplist distutils
+USE_PYTHON=	autoplist concurrent distutils
 
 OPTIONS_DEFINE=	PYGLET
 PYGLET_DESC=	Enable plotting support
