Index: math/py-igraph/Makefile
===================================================================
--- math/py-igraph/Makefile	(revision 378936)
+++ math/py-igraph/Makefile	(working copy)
@@ -12,6 +12,7 @@
 COMMENT=	High performance graph data structures and algorithms
 
 BUILD_DEPENDS=	${LOCALBASE}/lib/libigraph.so:${PORTSDIR}/math/igraph
+RUN_DEPENDS=	${LOCALBASE}/lib/libigraph.so:${PORTSDIR}/math/igraph
 
 USES=		python
 USE_PYTHON=	distutils autoplist
