Index: math/blitz++/Makefile
===================================================================
--- math/blitz++/Makefile	(revision 490640)
+++ math/blitz++/Makefile	(working copy)
@@ -11,7 +11,6 @@
 LICENSE=	GPLv3 BSD3CLAUSE
 LICENSE_COMB=	multi
 
-BROKEN_FreeBSD_12=	fails to compile: ../blitz/blitz.h:58:12: 'iostream.h' file not found
 BROKEN_FreeBSD_13=	fails to compile: ../blitz/blitz.h:58:12: 'iostream.h' file not found
 
 GNU_CONFIGURE=	yes
