Index: math/py-igraph/Makefile
===================================================================
--- math/py-igraph/Makefile	(revision 457506)
+++ math/py-igraph/Makefile	(working copy)
@@ -3,6 +3,7 @@
 
 PORTNAME=	igraph
 PORTVERSION=	0.7.0
+PORTREVISION=	1
 CATEGORIES=	math python
 MASTER_SITES=	http://igraph.org/nightly/get/python/
 PKGNAMEPREFIX=	${PYTHON_PKGNAMEPREFIX}
@@ -14,7 +15,7 @@
 BUILD_DEPENDS=	${LOCALBASE}/lib/libigraph.so:math/igraph
 
 USES=		python
-USE_PYTHON=	distutils autoplist
+USE_PYTHON=	distutils concurrent autoplist
 USE_GNOME=	libxml2
 
 .include <bsd.port.mk>
