Index: Mk/bsd.tex.mk
===================================================================
--- Mk/bsd.tex.mk	(revision 360535)
+++ Mk/bsd.tex.mk	(working copy)
@@ -464,11 +464,13 @@
 	${TEXMFVARDIR}/web2c/pdftex/etex.fmt \
 	${TEXMFVARDIR}/web2c/pdftex/pdfetex.log \
 	${TEXMFVARDIR}/web2c/pdftex/pdfetex.fmt \
-	bin/pdfetex
+	bin/pdfetex \
+	bin/etex
 TEX_FORMAT_PDFTEX_DIRS= \
 	${TEXMFVARDIR}/web2c/pdftex
 post-install-pdftex:
 	${LN} -sf pdftex ${STAGEDIR}${PREFIX}/bin/pdfetex
+	${LN} -sf pdftex ${STAGEDIR}${PREFIX}/bin/etex
 
 TEX_FORMAT_PTEX?= \
 	"ptex ptex - ptex.ini" \
