Index: multimedia/mplayer/Makefile.optvars
===================================================================
--- multimedia/mplayer/Makefile.optvars	(revision 433381)
+++ multimedia/mplayer/Makefile.optvars	(working copy)
@@ -116,7 +116,7 @@
 SKINS_RUN_DEPENDS=	mplayer-skins>=0:multimedia/mplayer-skins
 SKINS_IMPLIES=		GUI
 
-SMB_LIB_DEPENDS=	libsmbclient.so:net/samba-libsmbclient
+SMB_LIB_DEPENDS=	libsmbclient.so:net/samba43
 SMB_CONFIGURE_OFF=	--disable-smb
 
 SPEEX_LIB_DEPENDS=	libspeex.so:audio/speex
