Index: multimedia/ffmpeg/Makefile
===================================================================
--- multimedia/ffmpeg/Makefile	(revision 433381)
+++ multimedia/ffmpeg/Makefile	(working copy)
@@ -305,7 +305,7 @@
 SDL_CONFIGURE_OFF=	--disable-outdev=sdl --disable-sdl2
 
 # smbclient
-SMB_LIB_DEPENDS=	libsmbclient.so:net/samba-libsmbclient
+SMB_LIB_DEPENDS=	libsmbclient.so:net/samba43
 SMB_CONFIGURE_ENABLE=	libsmbclient
 SMB_IMPLIES=		GPL3
 
