Index: multimedia/mplayer2/Makefile
===================================================================
--- multimedia/mplayer2/Makefile	(revision 372681)
+++ multimedia/mplayer2/Makefile	(working copy)
@@ -26,7 +26,7 @@
 		TMPDIR="${WRKSRC}"
 CONFIGURE_ARGS=	--cc=${CC} \
 		--extra-cflags='-I${LOCALBASE}/include/portaudio2' \
-		--extra-libs='-lavresample' \
+		--extra-libs-mplayer='-lavresample' \
 		--extra-ldflags='-L${LOCALBASE}/lib/portaudio2' \
 		--mandir=${PREFIX}/man \
 		--enable-libavresample \
@@ -40,7 +40,7 @@
 		--disable-mpg123 \
 		--disable-musepack
 MAKE_ARGS+=	BINDIR=${STAGEDIR}${PREFIX}/bin \
-		CONFDIR=${STAGEDIR}${PREFIX}/etc/mplayer
+		CONFDIR=${STAGEDIR}${PREFIX}/etc/mplayer2
 
 MPLAYER2_SNAPSHOT_DATE=	2013-04-28
 MPLAYER2_PORT_VERSION=	2.0.${MPLAYER2_SNAPSHOT_DATE:S/-//g}
@@ -69,8 +69,6 @@
 
 SUB_FILES=	pkg-message
 
-CONFLICTS=	mplayer-1.*
-
 .include <bsd.port.options.mk>
 
 .if ${ARCH} == "sparc64"
@@ -306,11 +304,14 @@
 .endif
 
 post-install:
+	@${MV} ${STAGEDIR}${PREFIX}/bin/mplayer ${STAGEDIR}${PREFIX}/bin/mplayer2
+	@${MV} ${STAGEDIR}${PREFIX}/man/man1/mplayer.1 \
+		${STAGEDIR}${PREFIX}/man/man1/mplayer2.1
 	@${INSTALL_DATA} ${WRKSRC}/etc/example.conf \
-		${STAGEDIR}${PREFIX}/etc/mplayer/mplayer.conf.sample
+		${STAGEDIR}${PREFIX}/etc/mplayer2/mplayer.conf.sample
 .for conf in codecs input
 	@${INSTALL_DATA} ${WRKSRC}/etc/${conf}.conf \
-		${STAGEDIR}${PREFIX}/etc/mplayer/${conf}.conf.sample
+		${STAGEDIR}${PREFIX}/etc/mplayer2/${conf}.conf.sample
 .endfor
 
 .include <bsd.port.mk>
Index: multimedia/mplayer2/files/pkg-message.in
===================================================================
--- multimedia/mplayer2/files/pkg-message.in	(revision 372681)
+++ multimedia/mplayer2/files/pkg-message.in	(working copy)
@@ -2,7 +2,7 @@
 ==================================================================
 MPlayer's system-wide configuration dir is
 
-%%PREFIX%%/etc/mplayer
+%%PREFIX%%/etc/mplayer2
 
 ===================================================================
 If seeking in streams causes the playback volume to be reset to
Index: multimedia/mplayer2/pkg-plist
===================================================================
--- multimedia/mplayer2/pkg-plist	(revision 372681)
+++ multimedia/mplayer2/pkg-plist	(working copy)
@@ -1,6 +1,6 @@
-bin/mplayer
-etc/mplayer/codecs.conf.sample
-etc/mplayer/input.conf.sample
-etc/mplayer/mplayer.conf.sample
-man/man1/mplayer.1.gz
-@dirrmtry etc/mplayer
+bin/mplayer2
+etc/mplayer2/codecs.conf.sample
+etc/mplayer2/input.conf.sample
+etc/mplayer2/mplayer.conf.sample
+man/man1/mplayer2.1.gz
+@dirrmtry etc/mplayer2
