Index: net/samba43/Makefile
===================================================================
--- net/samba43/Makefile	(revision 437276)
+++ net/samba43/Makefile	(working copy)
@@ -155,6 +155,7 @@
 			--with-winbind \
 			--without-gettext \
 			--with-sendfile-support \
+			--disable-rpath \
 			${ICONV_CONFIGURE_BASE}
 # for libexecinfo: (so that __builtin_frame_address() finds the top of the stack)
 .if ${ARCH} == "amd64"
@@ -504,6 +505,7 @@
 
 post-install:
 			@${LN} -sf smb.conf.5.gz ${STAGEDIR}${PREFIX}/man/man5/smb4.conf.5.gz
+			@${LN} -sf samba4/libsmbclient.h ${STAGEDIR}${PREFIX}/include/libsmbclient.h
 .if ${PORT_OPTIONS:MDOCS}
 			@${MKDIR} ${STAGEDIR}${DOCSDIR}
 .	for doc in ${PORTDOCS}
Index: net/samba43/pkg-plist
===================================================================
--- net/samba43/pkg-plist	(revision 437276)
+++ net/samba43/pkg-plist	(working copy)
@@ -42,6 +42,7 @@
 %%AD_DC%%sbin/samba_upgradedns
 sbin/smbd
 sbin/winbindd
+include/libsmbclient.h
 include/samba4/charset.h
 include/samba4/core/doserr.h
 include/samba4/core/error.h
