Index: print/html2ps/Makefile
===================================================================
--- print/html2ps/Makefile	(revision 546304)
+++ print/html2ps/Makefile	(working copy)
@@ -14,7 +14,7 @@
 LICENSE=	GPLv2
 LICENSE_FILE=	${WRKSRC}/COPYING
 
-RUN_DEPENDS=	mogrify:graphics/ImageMagick6 \
+RUN_DEPENDS=	mogrify:graphics/ImageMagick7 \
 		p5-libwww>=0:www/p5-libwww \
 		weblint++:www/weblint++ \
 		paperconf:print/libpaper
