Index: print/pdftk/Makefile
===================================================================
--- print/pdftk/Makefile	(revision 490640)
+++ print/pdftk/Makefile	(working copy)
@@ -13,7 +13,6 @@
 
 LICENSE=	GPLv2
 
-BROKEN_FreeBSD_12=	gcj6: internal compiler error: Segmentation fault (program ecj1)
 BROKEN_FreeBSD_13=	gcj6: internal compiler error: Segmentation fault (program ecj1)
 
 # Get GCC version suffix without the dot in USE_GCC
