Index: security/sshit/Makefile
===================================================================
--- security/sshit/Makefile	(revision 402584)
+++ security/sshit/Makefile	(working copy)
@@ -10,8 +10,6 @@
 MAINTAINER=	ports@FreeBSD.org
 COMMENT=	Checks for SSH/FTP bruteforce and blocks given IPs
 
-BROKEN=		unfetchable
-
 RUN_DEPENDS=	p5-Unix-Syslog>=0:sysutils/p5-Unix-Syslog \
 		p5-IPC-Shareable>=0:devel/p5-IPC-Shareable \
 		p5-Proc-PID-File>=0:devel/p5-Proc-PID-File
