Index: security/cracklib/Makefile
===================================================================
--- security/cracklib/Makefile	(revision 375872)
+++ security/cracklib/Makefile	(working copy)
@@ -18,7 +18,7 @@
 DICTPATH=	${PREFIX}/libdata/${PORTNAME}
 DICTFILE=	${DICTPATH}/pw_dict
 GNU_CONFIGURE=	yes
-CONFIGURE_ARGS=	--with-default-dict=${DICTPATH}
+CONFIGURE_ARGS=	--with-default-dict=${DICTFILE}
 INSTALL_TARGET=	install-strip
 USES=		gettext libtool
 USE_LDCONFIG=	yes
