Index: security/kgpg-kde4/Makefile
===================================================================
--- security/kgpg-kde4/Makefile	(revision 372984)
+++ security/kgpg-kde4/Makefile	(working copy)
@@ -10,7 +10,7 @@
 MAINTAINER=	kde@FreeBSD.org
 COMMENT=	Encryption tool for KDE
 
-RUN_DEPENDS=	gpg2:${PORTSDIR}/security/gnupg
+RUN_DEPENDS=	gpg2:${PORTSDIR}/security/gnupg20
 
 USE_KDE4=	kdeprefix automoc4 kdelibs pimlibs
 USES=		cmake:outsource tar:xz
