Index: sysutils/wmcpuload/Makefile
===================================================================
--- sysutils/wmcpuload/Makefile	(revision 517746)
+++ sysutils/wmcpuload/Makefile	(working copy)
@@ -10,8 +10,6 @@
 MAINTAINER=	xride@FreeBSD.org
 COMMENT=	"lcd" dockapp for windowmaker, which displays the current cpuload
 
-BROKEN=		unfetchable
-
 USES=		tar:bzip2 xorg
 USE_XORG=	xpm
 GNU_CONFIGURE=	yes
