Index: sysutils/gnome-control-center/Makefile
===================================================================
--- sysutils/gnome-control-center/Makefile	(revision 413216)
+++ sysutils/gnome-control-center/Makefile	(working copy)
@@ -59,7 +59,7 @@
 OPTIONS_DEFINE_amd64=	CHEESE
 OPTIONS_DEFAULT_amd64=	CHEESE
 CUPS_LIB_DEPENDS=	libcups.so:print/cups \
-			libsmbclient.so:net/samba-libsmbclient
+			libsmbclient.so:net/samba43
 CUPS_CONFIGURE_ENABLE=	cups
 CHEESE_LIB_DEPENDS=	libcheese.so:multimedia/cheese
 CHEESE_CONFIGURE_WITH=	cheese
