Index: sysutils/wmcpuload/Makefile
===================================================================
--- sysutils/wmcpuload/Makefile	(revision 531763)
+++ sysutils/wmcpuload/Makefile	(working copy)
@@ -10,10 +10,6 @@
 MAINTAINER=	xride@FreeBSD.org
 COMMENT=	"lcd" dockapp for windowmaker, which displays the current cpuload
 
-BROKEN=		unfetchable
-DEPRECATED=	Broken for more than 6 months
-EXPIRATION_DATE=	2020-05-05
-
 USES=		tar:bzip2 xorg
 USE_XORG=	xpm
 GNU_CONFIGURE=	yes
