diff --git a/sysutils/accountsservice/Makefile b/sysutils/accountsservice/Makefile
index 86751efc375b..21fba0f5e942 100644
--- a/sysutils/accountsservice/Makefile
+++ b/sysutils/accountsservice/Makefile
@@ -25,9 +25,7 @@ INSTALL_TARGET=	install-strip
 
 # XXX HACK! this file isn't present on the pristine jails of the builders
 # and configure breaks without it. Beter solutions welcome.
-.if defined(PACKAGE_BUILDING)
 CONFIGURE_ENV+=	ac_cv_file__var_log_utx_log=yes
-.endif
 
 post-patch:
 	@${REINPLACE_CMD} -e 's|/etc/gdm/custom.conf|${LOCALBASE}/etc/gdm/custom.conf|g' \
