diff --git a/textproc/markdown/Makefile b/textproc/markdown/Makefile
index 4cea535a58f6..b00d9285126a 100644
--- a/textproc/markdown/Makefile
+++ b/textproc/markdown/Makefile
@@ -21,12 +21,12 @@ OPTIONS_DEFINE=	DOCS
 
 PORTDOCS=	Markdown-Readme.text
 
-PLIST_FILES=	bin/markdown
+PLIST_FILES=	bin/markdown.pl
 
 CONFLICTS_INSTALL=	discount
 
 do-install:
-	${INSTALL_SCRIPT} ${WRKSRC}/Markdown.pl ${STAGEDIR}${PREFIX}/bin/markdown
+	${INSTALL_SCRIPT} ${WRKSRC}/Markdown.pl ${STAGEDIR}${PREFIX}/bin/markdown.pl
 	@${MKDIR} ${STAGEDIR}${DOCSDIR}
 	${INSTALL_DATA} "${WRKSRC}/Markdown Readme.text" ${STAGEDIR}${DOCSDIR}/Markdown-Readme.text
 
