Index: www/swfdec-plugin/Makefile
===================================================================
--- www/swfdec-plugin/Makefile	(revision 508258)
+++ www/swfdec-plugin/Makefile	(working copy)
@@ -11,8 +11,6 @@
 MAINTAINER=	ports@FreeBSD.org
 COMMENT=	Flash rendering plugin
 
-BROKEN=		unfetchable
-
 LIB_DEPENDS=	libswfdec-0.8.so:graphics/swfdec
 
 USES=		libtool:keepla pkgconfig webplugin:gecko
