Index: x11-clocks/pclock/Makefile
===================================================================
--- x11-clocks/pclock/Makefile	(revision 512140)
+++ x11-clocks/pclock/Makefile	(working copy)
@@ -11,10 +11,6 @@
 MAINTAINER=	ports@FreeBSD.org
 COMMENT=	Simple analog clock program for the Window Maker dock
 
-BROKEN=		unfetchable
-DEPRECATED=	Unfetchable, unmaintained
-EXPIRATION_DATE=	2019-10-16
-
 WRKSRC=		${WRKDIR}/${PORTNAME}-${PORTVERSION}.orig/src
 
 USE_XORG=	x11 xau xdmcp xext xpm
