Index: x11/wdm/Makefile
===================================================================
--- x11/wdm/Makefile	(revision 531763)
+++ x11/wdm/Makefile	(working copy)
@@ -17,10 +17,6 @@
 MAINTAINER=	xride@FreeBSD.org
 COMMENT=	WINGs Display Manager; an xdm replacement
 
-BROKEN=		unfetchable
-DEPRECATED=	Broken for more than 6 months
-EXPIRATION_DATE=	2020-05-05
-
 LIB_DEPENDS=	libWINGs.so:x11-wm/windowmaker
 
 NOT_FOR_ARCHS=	sparc64
