Index: x11/xrestop/Makefile
===================================================================
--- x11/xrestop/Makefile	(revision 519661)
+++ x11/xrestop/Makefile	(working copy)
@@ -5,15 +5,14 @@
 PORTVERSION=	0.4
 PORTREVISION=	5
 CATEGORIES=	x11
+MASTER_SITES=	http://projects.o-hand.com/sources/${PORTNAME}/
 
 MAINTAINER=	bland@FreeBSD.org
 COMMENT=	X11 server side resource usage statistics monitor
 
-USES=		gmake pkgconfig ncurses xorg xorg-cat:app
+USES=		gmake pkgconfig ncurses xorg
 USE_XORG=	x11 xext xrender xt xres xorgproto
 GNU_CONFIGURE=	yes
-USE_GITLAB=	yes
-GL_COMMIT=	5e3d606e9e055c2442e90285b6fe7ffdc2f721b8
 
 CONFIGURE_ENV=	XLIBS="-L${LOCALBASE}/lib -lX11"
 LDFLAGS+=	-L${LOCALBASE}/lib
Index: x11/xrestop/distinfo
===================================================================
--- x11/xrestop/distinfo	(revision 519661)
+++ x11/xrestop/distinfo	(working copy)
@@ -1,3 +1,2 @@
-TIMESTAMP = 1575739363
-SHA256 (xorg/app/app-xrestop-5e3d606e9e055c2442e90285b6fe7ffdc2f721b8_GL0.tar.gz) = 2493395b196f319e811c4e9a128c223aaf8ef8a25a0a439687827a16ca3ba794
-SIZE (xorg/app/app-xrestop-5e3d606e9e055c2442e90285b6fe7ffdc2f721b8_GL0.tar.gz) = 18151
+SHA256 (xrestop-0.4.tar.gz) = 67c2fc94a7ecedbaae0d1837e82e93d1d98f4a6d759828860e552119af3ce257
+SIZE (xrestop-0.4.tar.gz) = 92308
