Index: x11/xrestop/Makefile
===================================================================
--- x11/xrestop/Makefile	(revision 517746)
+++ x11/xrestop/Makefile	(working copy)
@@ -10,8 +10,6 @@
 MAINTAINER=	bland@FreeBSD.org
 COMMENT=	X11 server side resource usage statistics monitor
 
-BROKEN=		unfetchable
-
 USES=		gmake pkgconfig ncurses xorg
 USE_XORG=	x11 xext xrender xt xres xorgproto
 GNU_CONFIGURE=	yes
