Index: x11/gnome2/Makefile
===================================================================
--- x11/gnome2/Makefile	(revision 372681)
+++ x11/gnome2/Makefile	(working copy)
@@ -103,12 +103,4 @@
 .endif
 .endif
 
-.if ${.CURDIR}==${MASTERDIR} || ${.CURDIR:T}=="gnome2-lite"
-.if ${PORT_OPTIONS:MXSCREENSAVER}
-RUN_DEPENDS+=	${LOCALBASE}/share/applications/xscreensaver-properties.desktop:${PORTSDIR}/x11/xscreensaver-gnome
-.else
-RUN_DEPENDS+=	gnome-screensaver:${PORTSDIR}/x11/gnome-screensaver
-.endif
-.endif
-
 .include <bsd.port.post.mk>
