Index: x11/kde4-runtime/Makefile
===================================================================
--- x11/kde4-runtime/Makefile	(revision 413216)
+++ x11/kde4-runtime/Makefile	(working copy)
@@ -14,7 +14,7 @@
 		libexiv2.so:graphics/exiv2 \
 		libwebp.so:graphics/webp \
 		libslp.so:net/openslp \
-		libsmbclient.so:net/samba-libsmbclient \
+		libsmbclient.so:net/samba43 \
 		libssh.so:security/libssh \
 		libqzeitgeist.so:sysutils/qzeitgeist
 # Explicitly depend on libssh to avoid interference with ssh.4 from
